codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/DebugPrintlnLog.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.codegen.scalasdk
+
+import com.lightbend.akkasls.codegen.Log
+
+case class DebugPrintlnLog(debugEnabled: Boolean) extends Log {
+  override def debug(message: String): Unit =
+    if (debugEnabled) println(s"[DEBUG] $message")
+  override def info(message: String): Unit =
+    if (debugEnabled) println(s"[INFO] $message")
```

Comment:
Who need a log framework? 😄

Reply:
Yeah... in Akka gRPC we have a hack to pass the sbt logger to the codegen (https://github.com/akka/akka-grpc/blob/e4abdfa60a5b1d3b7b7e816d05e859e16230db5b/sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala#L29), but perhaps we can do without here. We can pass fatal problems out, after all.